node/runtime/pangolin/src/lib.rs

```diff
@@ -702,7 +701,7 @@ sp_api::impl_runtime_apis! {
 			_lane_id: bp_messages::LaneId,
 			payload: ToPangoroMessagePayload,
 		) -> Option<Balance> {
-			estimate_message_dispatch_and_delivery_fee::<WithPangoroMessageBridge>(
+			bridge_runtime_common::messages::source::estimate_message_dispatch_and_delivery_fee::<WithPangoroMessageBridge>(
```

Comment:
@fewensa Do we use this to estimate our fee in bridger anymore, possible to be removed?